### PR TITLE
Added Grid Component

### PR DESCRIPTION
**Pull request by mansi2011**
Added Grid Component
grid

### CODE REVIEW

File: Components/Simplified-Grid/Grid.css [hunk 1]
```diff
@@ -0,0 +1,24 @@
+#grid-col {
+    display: grid;
+    grid-template-columns: auto auto auto;
+    width: 40rem;
```
Comment (AmishMishra11) [inline Components/Simplified-Grid/Grid.css:6]: you can use css variables as you have used 40rem in many places

File: Components/Simplified-Grid/Grid.css [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    padding: 10px;
+    text-align: center;
+  }
+
+
+#grid-row {
+   display: grid;
+   grid-template-columns: auto auto auto;
+   grid-template-rows: 50% 50%;
+   width: 40rem;
```
Comment (AmishMishra11) [inline Components/Simplified-Grid/Grid.css:23]: same as above

**Review by AmishMishra11 — APPROVED** [review]
Good folder structure just add one small change that I suggested then you can merge

**Review by shivanigangadharan — APPROVED** [review]
Code looks good, can merge.

**Pull request merged**